Client/Frontend/Widgets/OneLineTableViewCell.swift

```diff
@@ -0,0 +1,164 @@
+        setupNotifications(forObserver: self, observing: [.DisplayThemeChanged])
+        applyTheme()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private var defaultSeparatorInset: UIEdgeInsets {
+        return UIEdgeInsets(top: 0,
+                            left: UX.imageSize + 2 * UX.borderViewMargin,
+                            bottom: 0,
+                            right: 0)
+    }
+
+    private func setupLayout() {
+        separatorInset = defaultSeparatorInset
+        selectionStyle = .default
+
+        containerView.addSubviews(leftImageView,
+                                  titleLabel,
+                                  bottomSeparatorView)
+
+        contentView.addSubview(containerView)
+        bringSubviewToFront(containerView)
+
+        let containerViewTrailingAnchor = accessoryView?.leadingAnchor ?? contentView.trailingAnchor
+        let midViewLeadingMargin: CGFloat = shouldLeftAlignTitle ? 5 : 13
+
+        NSLayoutConstraint.activate([
+            containerView.topAnchor.constraint(equalTo: contentView.topAnchor,
+                                              constant: 8),
+            containerView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
+            containerView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor,
+                                                  constant: -8),
+            containerView.trailingAnchor.constraint(equalTo: containerViewTrailingAnchor),
+
+            leftImageView.centerYAnchor.constraint(equalTo: titleLabel.centerYAnchor),
+            leftImageView.leadingAnchor.constraint(equalTo: containerView.leadingAnchor,
+                                                  constant: 16),
+            leftImageView.widthAnchor.constraint(equalToConstant: 28),
+            leftImageView.heightAnchor.constraint(equalToConstant: 28),
+            leftImageView.trailingAnchor.constraint(equalTo: titleLabel.leadingAnchor,
+                                                    constant: -midViewLeadingMargin),
+
+            titleLabel.topAnchor.constraint(equalTo: containerView.topAnchor,
+                                            constant: 4),
+            titleLabel.bottomAnchor.constraint(equalTo: containerView.bottomAnchor,
+                                               constant: -4),
+            titleLabel.trailingAnchor.constraint(equalTo: containerView.trailingAnchor,
+                                                 constant: -8),
+
+            bottomSeparatorView.leadingAnchor.constraint(equalTo: titleLabel.leadingAnchor),
+            bottomSeparatorView.bottomAnchor.constraint(equalTo: containerView.bottomAnchor),
+            bottomSeparatorView.trailingAnchor.constraint(equalTo: containerView.trailingAnchor),
+            bottomSeparatorView.heightAnchor.constraint(equalToConstant: 0.7)
+        ])
+
+        selectedBackgroundView = selectedView
+        applyTheme()
```

Comment:
We're calling this here even though we're also calling it afterwards? Is there a reason?

Reply:
no I just didn't see in the clean up